root-packages/wush/build.sh

```diff
@@ -0,0 +1,30 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/coder/wush/
+TERMUX_PKG_DESCRIPTION="simplest & fastest way to transfer files between computers"
+TERMUX_PKG_LICENSE="CC0-1.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.4.1"
+TERMUX_PKG_SRCURL=https://github.com/coder/wush/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=77d5a912465d1e8ec478252a9a69a04d39af75a126ac9ed94823f33a60b3d8f9
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_post_get_source() {
+	termux_setup_golang
+	export GOPATH=$TERMUX_PKG_SRCDIR/go
+	export GOOS="android"
+	cd $TERMUX_PKG_SRCDIR/cmd/wush
+	go get
+	chmod +w $GOPATH -R
+}
+
+termux_step_make() {
+	cd $TERMUX_PKG_SRCDIR/cmd/wush
+	local _Commit=$(git ls-remote --tags https://hub.myxuebi.top/coder/wush.git | grep refs/tags/v0.4.1 | grep '\^{}' | awk '{print $1}')
```

Comment:
I think better to:
1. use github.com instead of mirror
2. use TERMUX_PKG_VERSION instead of hardcode version